roles/network_plugin/calico/tasks/check.yml

```diff
@@ -50,10 +50,14 @@
   run_once: True
   delegate_to: "{{ groups['kube_control_plane'][0] }}"
 
-- name: Check if calico ClusterInformation exists
-  command: "{{ bin_dir }}/calicoctl.sh get ClusterInformation"
-  register: clusterinformation_exists
+- name: Check if calico exists
```

Comment:
Is it better to change the name to `Check if calico ready`?

Reply:
It sounds good.